qualtricssurvey/qualtricssurvey.py

```diff
@@ -13,49 +13,105 @@
 
 class QualtricsSurvey(XBlock):
     """
-    Xbock for creating a Qualtrics survey.
+    Xblock for creating a Qualtrics survey.
     """
-
-    @staticmethod
-    def workbench_scenarios():
-        """
-        Gather scenarios to be displayed in the workbench
-        """
-        return [
-            ('Qualtrics Survey',
-             """<sequence_demo>
-                    <qualtricssurvey />
-                    <qualtricssurvey name="My First XBlock" />
-                </sequence_demo>
-             """),
-        ]
-
-    name = String(
-        default='Qualtrics Survey',
+    display_name = String(
+        default="Qualtrics Survey",
+        scope=Scope.settings,
+    )
+    survey_id = String(
+        default="Enter your survey ID here.",
```

Comment:
This should be done with a placeholder on the INPUT element, not as an actual default value.